internal/eval/eval.go

```diff
@@ -1737,7 +1738,8 @@ func evalMULTI(args []string, store *dstore.Store) []byte {
 // Every time a key in the watch list is modified, the client will be sent a response
 // containing the new value of the key along with the operation that was performed on it.
 // Contains only one argument, the query to be watched.
-func EvalQWATCH(args []string, clientFd int, store *dstore.Store) []byte {
+func EvalQWATCH(args []string, clientFd int, httpOp bool, httpClientWriter http.ResponseWriter,
```

Comment:
can we use io.Writer here? The idea is to keep the core decoupled from the transport layer if possible. I would imagine websockets to also need something similar?